### ISSUE TITLE
Can't save page when block has property with name "type"

### ISSUE THREAD

**Issue by mob8607** [sulu/sulu#2147]
| Q | A |
| --- | --- |
| Bug? | yes |
| New Feature? | no |
| Sulu Version | 1.1.10 |
| Browser Version | Chrome |
#### Actual Behavior

It's not possible to save the page
#### Expected Behavior

It should be possible to save the page
#### Steps to Reproduce

Create a template with block-property and add a property with name "type":

```
<block name="block"
       default-type="default"
       minOccurs="1"
       maxOccurs="10">
    <meta>
        <title lang="de">Block</title>
        <title lang="en">Block</title>
    </meta>

    <types>
        <type name="default">
            <meta>
                <title lang="de">Block Typ Standard</title>
                <title lang="en">Block Type Default</title>
            </meta>

            <properties>
                <property name="type" type="text_line">
                    <meta>
                        <title lang="de">Typ</title>
                        <title lang="en">Type</title>
                    </meta>
                </property>
            </properties>
        </type>
    </types>
</block>
```

**Comment by wachterjohannes** [comment]
type is a reserved name in the blocks. maybe we should prefix that or at least throw an error if a template contains a "type" named property in blocks.

**Comment by danrot** [comment]
The cleanest but BC breaking solution would be remove the namespace `i18n` for localizations, so that we could namespace properties like that with `sulu`. However, that's not a good idea currently. But prefixing like @wachterjohannes mentioned is the same kind of BC break. So I'd say throwing an error and also implementing something like suggested in https://github.com/sulu-io/sulu-standard/issues/636 would be the solution for now.

**Comment by wachterjohannes** [comment]
i would also validate that corectly and throw an error if a type property is used inside the block

**Comment by danrot** [comment]
Fixed in #5026
